samples/hosting/docker/sample.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Hosting endpoints in Docker Linux containers
+summary: Hosting multiple endpoints in several Docker Linux containers managed by Docker Compose
+reviewed: 2017-09-25
+component: Core
+tags:
+- Hosting
+related:
+- nservicebus/hosting
+---
+
+This sample demonstrates how to use Docker Linux containers to host NServiceBus endpoints communicating over the RabbitMQ transport.
+
+## Prerequisites
+
+This sample requires that the following tools are installed:
+
+  - [.NET Core 2.0 SDK](https://www.microsoft.com/net/download/core)
+  - [Docker Community Edition](https://www.docker.com/community-edition) or higher
+
+NOTE: Container that runs the RabbitMQ broker is a Linux container and [Docker for Windows also needs to be configured to use Linux containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
```

Comment:
**The** container.

Also this seems like another prerequisite, why not just a bullet?

- If using Windows, [configure Docker to use Linux containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers) to support the Linux-based RabbitMQ container

(If that's still accurate)